tests/test_go.py

```diff
@@ -44,6 +44,10 @@ def test_go_version(auto_container):
             repository_url="https://github.com/weaveworks/kured.git",
             repository_tag="1.13.2",
             build_command="make bootstrap-tools kured && go test -race ./...",
+            marks=pytest.mark.xfail(
+                condition=LOCALHOST.system_info.arch != "x86_64",
+                reason="Currently broken on arch != x86_64",
```

Comment:
would be nice to add a link / reference so that people can check when to remove it

Reply:
I've added a reference to https://github.com/kubereboot/kured/issues/823